rpi-eeprom-update

```diff
@@ -262,10 +262,6 @@ checkDependencies() {
       FIRMWARE_IMAGE_DIR="${FIRMWARE_IMAGE_DIR}-${BOARD_REVISION}"
    fi
 
-   if ! command -v vl805 > /dev/null; then
```

Comment:
Would it be appropriate to add this check to _just_ the places in the code where the `vl805` program actually gets executed?